source/reference/cutover-process.txt

```diff
@@ -16,16 +16,33 @@ You can finalize a migration and transfer your application
 workload from the source to the destination cluster using the
 :ref:`mongosync <c2c-mongosync>` cutover process.
 
-``mongosync`` should remain active until it reaches the
-:ref:`COMMITTED <c2c-state-committed>` state. This allows ``mongosync`` to sync
-any additional writes that occur during the migration.
-
 .. note::
 
    Before you switch your application workload to the
    destination cluster, you should always verify a successful sync.
    For more information, see :ref:`c2c-verification`.
 
+``mongosync`` should remain active until it reaches the
+:ref:`COMMITTED <c2c-state-committed>` state. This allows ``mongosync`` to sync
+any additional writes that occur during the migration.
+
+.. warning::
+
+   If you do not properly cutover from your source to your
+   destination, you may face the following issues:
+
+   - Unstructured or unneeded data on the destination cluster
+   - Commit failures with no option to rollback the sync
```

Comment:
I think we can delete the "commit failures with no option [...]" bullet in favor of the unrecoverable and undiagnosable errors bullet below